spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb

```diff
@@ -294,16 +298,81 @@ EOF
 
 }
 
+create_remote() {
```

Comment:
Can we reuse some parts? 
I think you copied  things from `create()`. So maybe it is possible to call this from "create" to prevent duplicate code.

Otherwise we forget about it and change only 1 place and forget the other.

Reply:
i put this all in create now and removed create_remote